apps/nextjs-website/src/lib/strapi/tutorial.ts

```diff
@@ -0,0 +1,61 @@
+import * as t from 'io-ts/lib';
+import * as tt from 'io-ts-types';
+import * as qs from 'qs';
+import { fetchFromStrapi } from './fetchFromStrapi';
+import { DateFromISOString } from 'io-ts-types/lib/DateFromISOString';
+import { MediaCodec } from './codecs/MediaCodec';
+import { NullToUndefinedCodec } from './codecs/NullToUndefinedCodec';
+import { BlocksContentCodec } from './codecs/BlocksContentCodec';
+import { PaginationCodec } from './codecs/PaginationCodec';
+import { ProductCodec } from './codecs/ProductCodec';
+import { RelatedLinksCodec } from './codecs/RelatedLinksCodec';
+
+const BannerLinkCodec = t.strict({
+  id: t.number,
+  title: t.union([NullToUndefinedCodec, t.string]),
+  body: t.union([NullToUndefinedCodec, BlocksContentCodec]),
+});
+
+export const TutorialCodec = t.strict({
+  id: t.number,
+  attributes: t.strict({
+    title: t.string,
+    slug: t.string,
+    content: BlocksContentCodec,
```

Comment:
Maybe we should refactor both BE and FE and replace this BlockContentCodec with Parts to manage code blocks and other parts
![image](https://github.com/pagopa/developer-portal/assets/39835990/ae8ba1c0-da27-4d8a-baf1-11b1c4deb752)

Reply:
I've inserted a new story about this activity in the project's backlog:
https://pagopa.atlassian.net/browse/DEV-1742